api/notd/manager.py

```diff
@@ -251,10 +261,10 @@ async def update_token_metadata(self, registryAddress: str, tokenId: str) -> Non
             retrievedTokenMetadata = await self.tokenMetadataProcessor.retrieve_token_metadata(registryAddress=registryAddress, tokenId=tokenId)
         except TokenDoesNotExistException:
             logging.info(f'Failed to retrieve non-existant token: {registryAddress}: {tokenId}')
-            return
+            return self.set_default(registryAddress=registryAddress, tokenId=tokenId)
```

Comment:
this is not right - this function should not return anything. you should be changing this function structure so that the default gets saved into the database for these tokens